test/test-trace-agent.js

```diff
@@ -34,17 +34,13 @@ var assert = require('assert');
 var config = require('../config.js');
 var file = require('../src/trace-agent.js');
 var SpanData = require('../src/span-data.js');
-// TODO: This line will silently fail if a logger is not supplied as
-//       the second argument.
```

Comment:
I think this TODO refers to `TraceAgent` not accounting for the second argument not being present. I know `TraceAgent` isn't exposed publicly but it would still be worth addressing this so future tests don't also cause the agent to silently fail.